src/lib/markup/stimuli.js

```diff
@@ -1,6 +1,11 @@
 const baseStimulus = (element, prompt=false) => {
-  const class_ = (prompt) ? 'main-prompt': 'main'
-  return `<div class=${class_}>${element}</div>`
+  return (
+    `<div class='center_container'>
```

Comment:
does the logic on main/main prompt being deleted mess anything else up? `main-prompt` and `center_container` have very similar styling - why not use what was here?

Reply:
Main prompt was putting the stimulus at the top of the screen rather than the center. I wasn't sure how else to center the stimulus (the letters)

Reply:
is main-prompt used anywhere else? should it be deleted now if you aren't using it here?

Reply:
no, it's not used anywhere else. (Other than where it's defined in the app.css file). I don't have a strong opinion

Reply:
that's fine, but can you go through the css file and make sure that we're using what's in there/ delete anything that's unused?